Core/Src/compute.c

```diff
@@ -98,12 +102,14 @@ int compute_send_charging_message(uint16_t voltage_to_set, uint16_t current_to_s
 	} charger_msg_data;
 
 	charger_msg_data.charger_voltage = voltage_to_set * 10;
-	charger_msg_data.charger_current = current_to_set * 10;
+	// Override charging to charge at 3.5A (or 0 if charging disabled)
+	charger_msg_data.charger_current = current_to_set == 0 ? 0 : 35;
+	printf("Current Requested: %d \r\n", charger_msg_data.charger_current);
```

Comment:
Note that with good charger this should be 5 amps, we lowered to 3.5 for Burlington. Not sure what capabilities are at comp